extensions/x-ms-enum-flags.md

```diff
@@ -0,0 +1,66 @@
+# OpenAPI Extension: x-ms-enums-flags
+
+This extension enables API providers to specify an enum as a bitwise(flagged) enum as well as the format of serialization of the multiple selected values.
+
+## x-ms-enums-flags object
+
+### Properties
+
+Properties are optional unless specified otherwise.
+
+| name | type | description |
+|---|---|---|
+| isFlags | boolean | Whether or not the enum is bitwise(flagged). Default is `false`(same as not specifying the extension) |
+| style | string | The serialization format of the selected enum values. Default is `simple` |
```

Comment:
I'm not sure `style` is the right attribute here.  Also, as `simple` is both the only supported value and the default value, why don't we just not define this property until we know we need it.

Reply:
I think this is a fair suggestion. If it's okay, we can drop this property and leave its definition for https://github.com/microsoft/kiota/issues/3238. cc @baywet

Reply:
we're not using it for the time being because passing the value would mean a breaking change for go and dotnet. I'm ok for dropping it for now (and cleaning up the code in kiota)

Reply:
Will create a PR to cleanup in Kiota